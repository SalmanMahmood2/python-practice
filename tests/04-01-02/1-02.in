Daniel
male
Ella
female
Sky
other
Mike
male
Jade
female
Riley
other
