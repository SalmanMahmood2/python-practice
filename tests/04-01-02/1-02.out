Daniel
Mike
Ella
Jade
Sky
Riley